src/docs/asciidoc/antora.yml

```diff
@@ -0,0 +1,16 @@
+name: ROOT
+title: The JHipster Mini-Book
+version: master
```

Comment:
Change we change this to main if it refers to a GitHub branch? I no longer use master as a branch name.

Reply:
Nice catch, you're correct I will update the branch name

Reply:
I've restored the value `master` and explain why in the `antora.yml` file.